moto/ssm/models.py

```diff
@@ -1,4 +1,5 @@
 import re
+import uuid
```

Comment:
Minor nit: 

users can seed Moto, so that 'random' identifiers are no longer random anymore. That means that for every 'random' identifier, we have to use a seeded generator:
```
from moto.moto_api._internal import mock_random
mock_random.uuid4()
```

Docs if you're interested: http://docs.getmoto.org/en/latest/docs/configuration/recorder/index.html#deterministic-identifiers